### DISCUSSION TITLE
To Linux and Mac users

### DISCUSSION THREAD

**Discussion by IceHacks** [IceHacks/SurvivCheatInjector#784] [Announcements]
Windows users probably have noticed that I have pushed a new update (3.0.7) fixing several bugs. Linux and Mac users will not get this update until next month. This is due to poor optimization in my build program that wasted resources. Once Github regenerates those resources (next month) I can start making builds for you non-Windows-compliant people. Thanks.

**Comment by AsiaGuy** [comment]
Very noice, i can play on other pc >:)

**Comment by IceHacks** [comment]
Also, I do not think auto-updatw will work for those devices at the moment. I will post all the here just in case it doesn't.

**Comment by IceHacks** [comment]
Expect a 3.0.8 release today Mac+Linux users. I am interested if the autoupdate will work.

**Reply by AsiaGuy** [reply]
yes, it worked thank you

**Comment by santosmo** [comment]
> Windows users probably have noticed that I have pushed a new update (3.0.7) fixing several bugs. Linux and Mac users will not get this update until next month. This is due to poor optimization in my build program that wasted resources. Once Github regenerates those resources (next month) I can start making builds for you non-Windows-compliant people. Thanks.
